### ISSUE TITLE
Who can be a member?

### ISSUE THREAD

**Issue by nothingismagick** [sfosc/sfosc#64]
In PR #54 @Beanow wrote that it was imperative that people participate in a discussion here in this repository before being granted member status. 

I didn't read that part that you just cited as demanding participation HERE in sfosc. Maybe I am wrong, but I think it is exactly this kind of engaged person that @vsoch is who should be welcomed here - and will definitely bring a huge network and vast experience to bear.

Merely blocking because of a formality (that seems rather trivial) seems heavy handed. Which is why I turned that discussion in a PR into its own issue for discussion.

And finally, discussing that issue here would resolve the blocker of involves:vsoch - thus opening the doors to their membership.

**Comment by vsoch** [comment]
Thanks for opening the issue @nothingismagick, I had seen the note in the README about participation:

> If you want to join the project as a voting member (which pretty much just means you get to vote for the project leader), add yourself to the membership file.

and (incorrectly) thought that the order of operations was to be added as a member, and then be included in discussion, voting, etc. I misunderstood that this comes only after contribution to this specific (repo) and not being a general open source person. think it would be good to clarify this detail. Thanks a bunch!

**Comment by Beanow** [comment]
Thank you both for this issue.
You're right I had interpreted it as: anyone is free to join discussions, suggestions, but only when it comes to being a voting member you need to have participated on this repo.

I'm currently being more formal, because I literally received access hours ago (see #60) without member input, and don't feel like that grants me the right to make judgement calls. So I try to adhere to the existing rules or clear consensus. Please bear with me as it becomes more clear what my role should be and keep sending me any and all feedback like this.

Reading @vsoch's take, I think it would be good to make it more clear what the order is.

In my opinion all participation should be open by default without membership requirement. With voting being the only participation that currently requires membership. What do you think?

And for what "proof of engagement with the community" means, I think it should be opened to outside this repo. But do you think it should be specific to _sustainable_ open source, or include open source in general?

**Comment by vsoch** [comment]
With respect to needing to prove oneself to be a member, I completely agree @Beanow. I'd maybe even set some criteria like "participates for X months, then is granted membership. Even if someone participates outside of the repository, it's still reasonable to prove oneself with a different community. If there are special cases, then it's going to get harder to quantify the level of outside participation that is good enough - better to stick with the same criteria for all! :)

**Comment by Beanow** [comment]
Hmm, the details of a participation requirement I think depend on why it's there in the first place.

If it's purely as protection against people gaming the voting system by creating new accounts, any account that has seen activity over a few months should be acceptable.

If it's to ensure members have sufficient experience to make informed votes, I think that's a poor indicator. If that's the goal, I'd go with a proper application with case-by-case evaluation.

Looking at previous membership PRs #10 #11 #26 #29 that's not what appears to be happening, so I propose we keep things open to anyone with 6 months of activity, regardless of frequency / quality / role.

Does that seem reasonable?

**Comment by themightychris** [comment]
I took a stab at mocking what I think a membership PR could look like #68 -- documenting with some links and a quick overview what your stake is in all this

**Comment by Beanow** [comment]
Thank you @themightychris, I much like this free format mention of relevant work should we choose a proof of engagement / stake / experience version. It would still be nice to use this PR as a chance to mention otherwise. Though unfortunately it'll be hidden once merged.

Meanwhile I believe regardless of which way it goes, yours and @vsoch's application meet every requirement I can think of. So I'm happy to accept even before we've fixed the language and details of the requirements.

As for those requirements, I'd like to discuss a chicken or egg situation. In my case diving into sustainability more devoutly is recent and SFOSC already has most of my public contributions towards this. There may be other people either uncomfortable with introductions and linking work right off the bat, or just now intending to join the debate. Accepting them may be good encouragement.

**Comment by Beanow** [comment]
#77 is an attempt to address this issue. Can you help review it?

**Comment by Beanow** [comment]
Thanks all! I think #73 and #77 have addressed this issue now so I'll close this.
